src/components/about-modal/about-modal.tsx

```diff
@@ -8,6 +8,7 @@ import {
   TextListVariants,
 } from '@patternfly/react-core';
 import { detect } from 'detect-browser';
+import { useEffect, useState } from 'react';
 import * as React from 'react';
```

Comment:
ideally, merge these into 

```suggestion
import React, { useEffect, useState } from 'react';
```

(but we can do it in a separate PR)